clang/lib/Sema/SemaTemplate.cpp

```diff
@@ -759,13 +759,11 @@ Sema::BuildDependentDeclRefExpr(const CXXScopeSpec &SS,
       TemplateArgs);
 }
 
-bool Sema::DiagnoseUninstantiableTemplate(SourceLocation PointOfInstantiation,
-                                          NamedDecl *Instantiation,
-                                          bool InstantiatedFromMember,
-                                          const NamedDecl *Pattern,
-                                          const NamedDecl *PatternDef,
-                                          TemplateSpecializationKind TSK,
-                                          bool Complain /*= true*/) {
+bool Sema::DiagnoseUninstantiableTemplate(
+    SourceLocation PointOfInstantiation, NamedDecl *Instantiation,
+    bool InstantiatedFromMember, const NamedDecl *Pattern,
+    const NamedDecl *PatternDef, TemplateSpecializationKind TSK,
+    bool Complain /*= true*/, bool *Unreachable) {
```

Comment:
@AaronBallman I don't think I have seen annotation before, annotating the default value in a function definition.

Are there tools that verify this like we have for [argument comments?](https://clang.llvm.org/extra/clang-tidy/checks/bugprone/argument-comment.html) 

Otherwise I don't think these make as much sense, although I sympathize with the sentiment behind them.

Reply:
On a tangental note, I think having Clangd show the default argument would help to workaround the lack of these annotations, so I filed #130571.

I don't think we have any tools to check these.

Reply:
Yeah, might as well drop it here otherwise it is a potential to get out of sync in the future.